.snyk

```diff
@@ -4,5 +4,6 @@ ignore: {}
 patch: {}
 exclude:
   global:
+    - vendor/github.com/Azure/go-autorest/Gopkg.lock
     - vendor/
```

Comment:
Before we merge this let's figure out why it's not working, since you should only have to specify a folder, not a subfolder. This list should be minimal so lets look at docs and then if that doesn't answer, reach out to security team on why it's not working.

Reply:
@sheldonhull security team is aware of this issue. Alexa Kessler did her homework on this issue and verified with with Snyk representatives. She would be the source of authority for any further/specific details. 
This PR's primary function is to test if including this single file in the .snyk policy file "exclude" directive will eliminate the snyk web UI from reporting these "High" level vulnerabilities. FYI, the snyk cli does not report the vulnerabilities being addressed by this PR.

Reply:
@pacificcode I'm discussing with Alexa for some context before merging since there's some research I did on this prior that might have relevance. Once I confirm with her I'll either approve or update here with more context, so leave this open temporarily  and I'll resolve it once I get some details to add. Didn't want you to think I'd forgotten :-)

Reply:
same update from 2 weeks ago. I did bump the internal conversation and will update once I have something more, or you can bump that thread as well. 🙇🏻